README.md

```diff
@@ -20,6 +20,16 @@ Options:
   -h --help         Show this.
 ~~~
 
+It also can be used inside node code
+
+~~~ text
+import tapDifflet from 'tap-difflet';
```

Comment:
I used ES6 stuff in my example before, but we should probably use node-friendly syntax here instead:

```js
var tapDifflet = require('tap-difflet');

var formatter = tapDifflet({
  pessimistic: true // Only output failed tests. `false` by default.
});
```

I also added a small comment to clarify the usage inline.